src/commands/duckrelated.py

```diff
@@ -0,0 +1,110 @@
+import os
+import random
+import aiohttp
+from discord import app_commands, Interaction, Embed
+from discord.ext import commands
+from dotenv import load_dotenv
+from typing import Optional, List
+
+from constants.duck_data import DUCK_FACTS, DUCK_JOKES
```

Comment:
I suggest breaking imports up into clear blocks of standard, third party, and project-specific imports as suggested in [PEP 8](https://peps.python.org/pep-0008/#imports).
```suggestion
import os
import random
import aiohttp

from discord import app_commands, Interaction, Embed
from discord.ext import commands
from dotenv import load_dotenv
from typing import Optional, List

from constants.duck_data import DUCK_FACTS, DUCK_JOKES
```

Reply:
I'll just note that `typing` is a standard library import, while `aiohttp` is third party